builtin/mainmenu/common.lua

```diff
@@ -81,8 +55,13 @@ function order_favorite_list(list)
 	return res
 end
 
---------------------------------------------------------------------------------
-function render_serverlist_row(spec, is_favorite)
+local function render_client_count(n)
+	if     n > 99 then return '99+'
```

Comment:
`if     n > 999 then return '999'`
_My servers easily cope with 120+ players. It was also 150+ several times!_

Reply:
Well, in fact that is only used to render the possible client count AFAIK.

Reply:
And the servers look like 99+ / 99+
instead of 110 / 200.
Table cell size? Same.
Informative? Worse.
Inexperienced player thinks that on his favorite server the maximum number of players!